src/en/pages/index.md

```diff
@@ -60,31 +60,58 @@ date: 'git Last Modified'
   <gcds-grid tag="ul" columns="1fr" columns-tablet="1fr 1fr">
-    <li class="list-none bg-white p-450 b-radius-md">
-      <h3 class="mb-300">
-        <gcds-link href="{{ links.getInvolved }}">Get involved</gcds-link>
-      </h3>
-      <p>Check out what we're working on.</p>
-    </li>
-    <li class="list-none bg-white px-175 py-450 b-radius-md">
-      <h3 class="mb-300">
-        <gcds-link external href="{{ links.releaseNotes }}">Release notes</gcds-link>
-      </h3>
-      <p>Read about the latest additions.</p>
-    </li>
+    <gcds-card
+      href="{{ links.registerDemo }}"
+      card-title="Attend a demo"
+      card-title-tag="h3"
+      description="Learn how GC Design System can work for you and your team and get your questions answered."
+      role="listitem"
+    ></gcds-card>
+    <gcds-card
+      href="{{ links.accessibility }}"
+      card-title="Find out about accessibility"
+      card-title-tag="h3"
+      description="Review accessibility practices and checks within the design system."
+      role="listitem"
+    ></gcds-card>
+    <gcds-card
+      href="{{ links.getInvolved }}"
+      card-title="Get involved"
+      card-title-tag="h3"
+      description="Check out what the team’s working on and how you can contribute."
+      role="listitem"
+    ></gcds-card>
+    <gcds-card
+      href="{{ links.releaseNotes }}"
+      card-title="Catch up on releases"
+      card-title-tag="h3"
+      description="Read about the latest additions and features."
+      role="listitem"
+    ></gcds-card>
   </gcds-grid>
 </article>
 
 <article class="py-450">
-  <h2 class="mb-300">Featured component</h2>
-
-  <h3 class="mb-300">Card</h3>
-  <p class="mb-450">A card is a box containing structured, actionable content on a single topic.</p>
-  <img class="d-block mb-300" src="../../images/common/components/preview-card.svg" alt="The card component shows a box containing a rectangular image in the top half. Immediately below is a blue rectangular bar running across three quarters of the box representing the card title. Two longer grey bars are stacked just below the blue bar, representing the card description and context area/metadata sections." />
-  <gcds-link href="{{ links.card }}">Read more about card</gcds-link>
+  <h2 class="mb-300">Featured components</h2>
 
-  <h3 class="mt-600 mb-300">Date input</h3>
-  <p class="mb-450">A date input is a space to enter a known date.</p>
-  <img class="d-block mb-300" src="../../images/en/components/preview/preview-date-input.svg" alt="Thick grey lines at the top represent the legend and hint text. A white box with a thin grey border and a grey line and grey arrow inside of it represents the select input for the month. Beside this are two smaller white boxes with thin grey outlines representing the text inputs for the day and year." />
-  <gcds-link href="{{ links.dateInput }}">Read more about date input</gcds-link>
+  <gcds-grid tag="ul" columns="1fr" columns-tablet="1fr 1fr">
```

Comment:
```suggestion
  <gcds-grid tag="ul" columns="1fr" columns-tablet="1fr 1fr" columns-desktop="1fr 1fr 1fr">
```

Thank you for adding in the cards, looks much nicer :) What do you think about slightly reducing the card width by creating a 3 columns grid and leaving the 3rd column empty?

Reply:
I like it :)